students/ivan/ejercicios-funciones-24-01-2022/script-2.js

```diff
@@ -0,0 +1,216 @@
+        return 'true'
+
+    }else{
+        return `false`
+    }
+}
+document.write(compare(a,b));
+document.write('<br>');
+
+
+/*
+3. Crea una función que dado un numero por 
+prompt, devuelva el número al revés por pantalla, 
+usar function expression.
+*/
+a = prompt('Ingrese un número');
+
+const numberInvertido = function(a){
+    
+    let b= '';    
+    for(let i=0; i<=a.length;i++){
+        b += a.charAt((a.length)-i);
+        
+    }
+    return parseInt(a);
+
+}
+document.write(numberInvertido(a));
+
+
+/*
+
+Resolución de clase. Trabajamos con numbres y no con strings
+*/
+const reverse = function(n){
+    let currentNumber = n;
+    let reverseNumber = 0;
+    while(currentNumber>0){
+        reverseNumber = reverseNumber*10 + (currentNumber%10);
+        currentNumber = Math.floor(currentNumber/10);
+
+    }
+    return reverseNumber;
+}
+
+
+
+/*
+4. Escribir varias funciones que dada las medidas
+ por prompt(radio, lados, etc) que calculen área y 
+ perímetro de diferentes figuras geométricas (Rectángulo, 
+    Cuadrado, triángulo y Círculo), para el circulo usaremos 
+    rectangulo y cuadrado usaremos arrow function.
+*/
+
+const areaSquare = a => `<p>El área del cuadrado de lado ${a} es ${a*a}</p>` ;
+
+function areaTriangle(a,b){
+    return `<p>El área del triángulo de lados ${a} y ${b} es ${0.5*a*b}</p>`;
+}
+
+const areaCircle = a => `<p>El área del círculo de radio ${a} es ${Math.PI*a*a}</p>`;
+
+a = parseInt(prompt('indique el valor del lado del cuadrado'));
+b = parseInt(prompt('indique el valor de la altura del triángulo'));
+let c = parseInt(prompt('indique el valor de la base del triángulo'));
+let d = parseInt(prompt('indique el valor del radio del círculo'));
+
+document.write(areaSquare(a));
+document.write(areaTriangle(b,c));
+document.write(areaCircle(d));
+
+/*
+5. Crear una función calculadora que reciba como 
+parámetros: Dos operandos y la función que se quiera 
+utilizar para calcular(+,-,*) y devuelva el resultado, 
+los números y la función se pasaran por pormpt, usar 
+function expression.
+*/
+
+const calculator = function(a,b,c){
+    if(a==='+'){
+        return `El resultado de ${b} ${a} ${c} es ${b+c}`;
+    }else if(a==='-'){
+        return `El resultado de ${b} ${a} ${c} es ${b+c}`;
+    }else if(a==='*'){
+        return `El resultado de ${b} ${a} ${c} es ${b*c}`;
+    }else if(a==='/'){
+        return `El resultado de ${b} ${a} ${c} es ${b/c}`;
+    }
+}
+
+a = prompt('Indique la operación que quiere realizar. +,-,*,/');
+b = parseInt(prompt('Indique el valor del primer numero'));
+c = parseInt(prompt('Indique el valor del segundo número'));
+
+document.write(calculator(a,b,c));
+document.write(domainGithub('<br>'));
+
+/*
+6. Crear una función que dado una URL
+ (Se representa con un string) por prompt, 
+ me diga o no si pertenece al dominio de github, 
+ usar function declaration.
+*/
+
+function domainGithub (a){
+    if(a.includes('github')===true){

```

Comment:
you do not need comparation under booleans. they are booleans itself. This can be refactored like `if(a.includes('github'))`